### PR TITLE
Use a bigger hammer on errors from github.

### PR DESCRIPTION
**Pull request by coderanger**
Ping @balanced-cookbooks/chefs

### CODE REVIEW

File: libraries/balanced_user.rb [hunk 1]
```diff
@@ -33,7 +33,10 @@ def _ssh_keys
       if github_username
         github_keys = begin
           Chef::HTTP.new('https://github.com').get("#{github_username}.keys")
-        rescue Net::HTTPServerException
+        # Use a really big hammer, github being down shouldn't break things.
+        # The downside is that if github is down, it will yank your key, possibly
+        # leaving login unavailable. Not sure what to do about this right now.
+        rescue Exception
           ''
```
Comment (bninja) [inline libraries/balanced_user.rb:40]: make sense to log so u know failed rather.. i assume '' is silent?

Comment (bninja) [inline libraries/balanced_user.rb:40]: also maybe condition swallow on presence of local ssh keys (e.g. mine is actually on user resource .ssh_keys)

**Comment by bninja** [comment]
otherwise lgtm for now until we have tm 2 revisit

File: libraries/balanced_user.rb (hunk 1 above)
Comment (coderanger) [inline libraries/balanced_user.rb:40]: Not much point in alerting since it isn't actionable by us in any real way.

Comment (bninja) [inline libraries/balanced_user.rb:40]: ok

**Pull request merged**